travel/models.py

```diff
@@ -1,3 +1,129 @@
 from django.db import models
+from datetime import date,datetime
+from django.urls import reverse
+from django.contrib.auth.models import User
+from django.core.files.storage import FileSystemStorage
 
 # Create your models here.
+
+class Tour(models.Model):
+  """Mode representing a tour."""
+  title = models.CharField(max_length = 200)
+  destination = models.CharField(max_length = 200)
+  start_location  = models.CharField(max_length = 200)
+  date = models.CharField(max_length = 100)
+  content = models.TextField (max_length = 1000, help_text = "Enter Description ")
+  place = models.CharField (max_length = 200, help_text = "places in tour",null=True, blank=True)
+  rating = models.FloatField()
+
+  class Meta:
+    ordering = ['-rating']
+
+  def __str__(self):
+    """String for representing the Model object."""
+    return self.title
+
+  def get_absolute_url(self):
+    """Returns the url to access a particular tour instance."""
+    return reverse('', args=[str(self.id)])
+
+class Image(models.Model):
+  """Model representing Image"""
+  tour = models.ForeignKey('Tour', on_delete = models.CASCADE, null=True, blank=True)
```

Comment:
Dùng cascade rồi thì remove cho phép null/blank nha em
```suggestion
  tour = models.ForeignKey('Tour', on_delete = models.CASCADE)
```